pom.xml

```diff
@@ -66,8 +66,8 @@
         <jts.bundle.version>1.12_1</jts.bundle.version>
         <karaf.version>2.4.3</karaf.version>
         <lux.version>1.2</lux.version>
-        <opengis.bundle.version>${org.geotools.version}_1</opengis.bundle.version>
-        <org.geotools.bundle.version>${org.geotools.version}_1</org.geotools.bundle.version>
+        <opengis.bundle.version>8.4_1</opengis.bundle.version>
```

Comment:
I don't know. This `8.4_1` is left over when when I was messing with the geotools version to see if I could get GML3 support. It didn't work, but I forgot to switch the bundle versions back.